BDArmory/Control/MissileFire.cs

```diff
@@ -7827,7 +7844,7 @@ public bool GetLaunchAuthorization(Vessel targetV, MissileFire mf, MissileBase m
 
                 // Check that target is within maxOffBoresight now and in future time fTime
                 //launchAuthorized = Vector3.Angle((mlauncher != null && mlauncher.missileTurret) ? mlauncher.missileTurret.finalTransform.forward :
-                launchAuthorized = missile.maxOffBoresight >= 360 ? true : Vector3.Angle(missile.GetForwardTransform(), target - missile.transform.position) < (unguidedWeapon ? 1 : missile.maxOffBoresight * boresightFactor); // Launch is possible now
+                launchAuthorized = missile.maxOffBoresight >= 360 ? true : Vector3.Angle(missile.GetForwardTransform(), target - missile.transform.position) < (unguidedWeapon ? 5 : missile.maxOffBoresight * boresightFactor); // Launch is possible now
```

Comment:
Not sure who wrote this originally, but `A ? true : B` is just a more convoluted way of writing `A || B`.
Also, wasn't this changed in Josue's PR (https://github.com/BrettRyland/BDArmory/pull/581/files#diff-566cc047c45ccfbc4c424b3603a3348972f9bcffb04f0f9b93362ad87bbae609R7765)?

Reply:
Yes. I hadn't updated my PR yet to account for his stuff getting merged in, and that got changed back to the original 5deg at some point during testing, so it showed up here.